PIVET.py

```diff
@@ -2762,3 +2769,16 @@ def redraw_image(self):
     #app.geometry("800x600")
     app.resizable()
     app.mainloop()
+
+# TODO Use a Python style guide or a linter
+#   https://peps.python.org/pep-0008/
+#   https://google.github.io/styleguide/pyguide.html
+
+# TODO You need to write unit tests.

```

Comment:
Sounds painful for such a large application, but will try. Do you have any tests to suggest?